lib/datasource/remote/model/member_model.dart

```diff
@@ -15,6 +16,15 @@ class MemberModel extends Equatable {
     );
   }
 
+  factory MemberModel.fromTelosAccount(String account) {
+    return MemberModel(
+      account: account,
+      nickname: "",
+      image: "",
```

Comment:
create his object with the image asset already? We know which image we want to use right ?

Reply:
Done

Reply:
Makes sense only reason I didn't do that is that I had originally planned to do this totally differently.